pkg/query-service/app/querier/helper.go

```diff
@@ -18,7 +18,7 @@ import (
 	"go.uber.org/zap"
 )
 
-func prepareLogsQuery(ctx context.Context,
+func prepareLogsQuery(_ context.Context,
```

Comment:
why this change ? Is it only because we are not using it?

Reply:
because linter complained about it; build would have failed if it was used anywhere.